docs/GoTopButton.md

```diff
@@ -0,0 +1,39 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Go Top Button
+
+`GoTopButton` is a component of VTEX that when clicked, goes back to the top of the page. This Component can be imported and used by any VTEX app.
+
+## Configuration
+
+1. Import the `vtex.store-component` app to your theme's dependencies in the manifest.json;
+
+```json
+ "dependencies: {
+    "vtex.store-components": "3.x"
+  }
+```
+
+2. Add the `go-top-button` block into your page. Example:
+
+```diff
+  "store.home": {
+    "blocks": [
++     "go-top-button",
+    ]
+  },
+```
+
```

Comment:
Could you add a third step with the `go-top-button` being declared and configured? Such as: 

3. Then, declare the `go-top-button` block using its props stated in the table below: 

```json
code example
```